scss/mixins/_grid.scss

```diff
@@ -10,27 +10,26 @@
 }
 
 @mixin make-col-ready($gutter: $grid-gutter-width) {
-  position: relative;
+  // Add box sizing if only the grid is loaded
+  box-sizing: if(variable-exists(include-column-box-sizing) and $include-column-box-sizing, border-box, null);
   // Prevent columns from becoming too narrow when at smaller grid tiers by
-  // always setting `width: 100%;`. This works because we use `flex` values
+  // always setting `width: 100%;`. This works because we set the width
   // later on to override this initial width.
+  flex-shrink: 0;
```

Comment:
Hi all: while I understand the rational for dropping `position: relative`, can you comment on why you added `flex-shrink: 0`? If would be great if you could respond to the following discussion thread. Thanks!

- https://github.com/twbs/bootstrap/discussions/37951